commands-yml/parse.js

```diff
@@ -98,9 +98,39 @@ Handlebars.registerHelper('if_eq', function (a, b, opts) {
   }
 });
 
-Handlebars.registerHelper('base_url', function (fullUrl) {
+function getBaseHostname (fullUrl) {
   const baseUrl = url.parse(fullUrl);
   return baseUrl.hostname;
+}
+
+Handlebars.registerHelper('base_url', function (fullUrl) {
+  return getBaseHostname(fullUrl);
+});
+
+Handlebars.registerHelper('client_url', function (clientUrl) {
+  if (!clientUrl) {
+    return;
+  }
+
+  const createUrlString = function createUrlString (clientUrl, name = getBaseHostname(clientUrl)) {
+    return `[${name}](${clientUrl})`;
+  };
+
+  if (!_.isArray(clientUrl)) {
+    return createUrlString(clientUrl);
+  }
+
+  let urlStrings = [];
+  for (const item of clientUrl) {
+    for (let [key, value] of _.toPairs(item)) {
+      key = key.toLowerCase();
+      const urlStr = ['ios', 'android'].includes(key)
```

Comment:
should these magic strings be hardcoded? should we make it more flexible in case we'd like to have more options?

Reply:
I think that at the moment it is unnecessary, and it's not immediately obvious how it should best be done. So better solved when there is a need, rather than over-engineering it right now with a "solution" that might not do what we need.